simcon_project/conversation_templates/views/create_conversation_template.py

```diff
@@ -0,0 +1,50 @@
+import json
+from django.shortcuts import render
+from django.http import HttpResponse
+from django.contrib.auth.decorators import user_passes_test
+from django.views.decorators.csrf import ensure_csrf_cookie
+from conversation_templates.models import TemplateNodeChoice, TemplateNode, ConversationTemplate
+from users.models import Researcher
+
+def is_researcher(user):
+    return user.is_authenticated and user.is_researcher
+
+@user_passes_test(is_researcher)
+@ensure_csrf_cookie  # guarantees csrf cookie is generated even though html does not contain {% csrf_token %}
+def CreateConversationTemplateView(request):
+
```

Comment:
extra line please remove

Reply:
> A) Hook up to nav bar
> B) I would like the template name/Template description to be a pop up i dont love it at the bottom vut thats like if we have time

I'm not a huge fan either. I'll make a ticket for later